redhat-upgrade-tool.py

```diff
@@ -365,6 +368,18 @@ def handle_sigwinch(signum, frame):
         upgrade_boot_args()
         prep_boot(kernel, initrd)
 
+    # Check for available space in /boot/ needed for kernel and grub
+    # installation during the upgrade.
+    statvfs = os.statvfs("/boot")
+    avail_bytes = statvfs.f_frsize * statvfs.f_bavail
+    if avail_bytes < MIN_AVAIL_BYTES_FOR_BOOT:
+        reset_boot()
+        additional_mib_needed = \
+            (MIN_AVAIL_BYTES_FOR_BOOT - avail_bytes) / (1024 * 1024)
```

Comment:
you should use `2**20` instead of `1024 * 1024` as Lojza pointed. It is more readable to me. But it is just detail.